README.rst

```diff
@@ -36,3 +36,42 @@ Run tests:
 
   $ cd bluesky-kafka
   $ pytest
+
+Run a Mongo Consumer Group
+--------------------------
+
+Create a conda environment:
+
+::
+
+  $ conda create -n consumers python=3.8
+  $ conda activate consumers
+
+Install packages:
+
+::
+
+  $ pip install bluesky-kafka supervisor
+
+Setup environment variables:
+mongo_uri reference: https://docs.mongodb.com/manual/reference/connection-string/
+bootstrap_servers: comma-separated list of brokers.
+
+::
+
+  $ export BLUESKY_MONGO_URI="mongodb://username:password@machine1:port1,machine2:port2,machine3:port3
+  $ export KAFKA_BOOTSTRAP_SERVERS="machine1:9092, machine2:9092, machine3:9092"
+
+Update the bluesky_kafka/supervisor/supervisord.conf file with the correct path for your installation.
+
+Start the consumer processes:
+
+::
+
+  $ supervisord -c bluesky_kafka/supervisor/supervisord.conf
```

Comment:
If there are more processes managed by `supervisor` I think we will want a centralized configuration like the one set up by Dan at SST with `supervisord.conf` in `/etc/supervisor` and a configuration file for each process in `/etc/supervisor/conf.d`. The next comment has a real explanation.

Reply:
I had some more ideas about keeping supervisor configuration in repositories. If we have a `<package_dir>/etc/supervisor/supervisord.conf` and `<package_dir>/etc/supervisor/conf.d` directory then during development we can use the `-c` you show in the README, and it also illustrates how to install the supervisor configuration  files.